tests/audio_block_format_objects_tests.cpp

```diff
@@ -1,146 +1,160 @@
 #define CATCH_CONFIG_ENABLE_CHRONO_STRINGMAKER
 #include <catch2/catch.hpp>
 #include "adm/elements/audio_block_format_objects.hpp"
+#include "helper/parameter_checks.hpp"
+
+// allows catch to print NamedType<std::chrono,V> on test failures, e.g. Rtime
+namespace Catch {
+  template<typename T, typename R, typename V>
+  using NamedChrono = typename adm::detail::NamedType<std::chrono::duration<T, R>, V>;
+
+  template<typename T, typename V>
+  using AdmTimeT = adm::detail::NamedType<adm::Time, T, V>;
+
+  // for e.g. Rtime
+  template<typename T, typename V>
+  struct StringMaker<AdmTimeT<T, V>> {
+    static std::string convert(AdmTimeT<T, V> const& value) {
+      auto time = value.get();
+      if (time.isNanoseconds()) {
+        auto ns = time.asNanoseconds();
+        return StringMaker< decltype(ns) >::convert(ns);
+      } else {
+        std::stringstream ss;
+        auto fractional = time.asFractional();
+        ss << fractional.numerator() << '/' << fractional.denominator() << " s";
+        return ss.str();
+      }
+    }
+  };
+  // for e.g. InterpolationLength
+  template<typename T, typename R, typename V>
+  struct StringMaker<NamedChrono<T, R, V>> {
+    static std::string convert( NamedChrono<T, R, V> const& value ) {
+      return StringMaker<std::chrono::duration<T,R>>::convert(value.get());
+    }
+  };
```

Comment:
There's some code related to this in adm_time_tests; it would be nice to have that in one place.